test/puppetlabs/puppetdb/http/nodes_test.clj

```diff
@@ -436,6 +436,15 @@
         (is (> (count (clojure.string/split-lines results))
                (count (clojure.string/split-lines normal-results))))))))
 
+(deftest-http-app query-with-explain-printing
+  [[version endpoint] endpoints
```

Comment:
We may want to move this test so that it hits a different endpoint to cover the issue we saw with the `munge-fn` erroring before. Because the nodes `nodes` endpoint uses `identity` for the `munge-fn` it wouldn't cause the same error seen when testing against the `fact-contents` endpoint. See [here](https://github.com/puppetlabs/puppetdb/blob/6.x/src/puppetlabs/puppetdb/query_eng.clj#L64) for the atom where the `munge-fn` is pulled out per entity type.

Reply:
I added a test in `resources_test.clj` because the munge function of `fact-contents` was valid for the query plan.